### ISSUE TITLE
Can't Compile SdK on ubuntu 20.04, wslv2

### ISSUE THREAD

**Issue by godreborn** [KallistiOS/KallistiOS#860]
I can compile this fine on 22.04, but on 20.04, this happens:

![image](https://github.com/user-attachments/assets/e0238190-b7bf-4c67-a16a-f9ac245dbe18)

it makes no sense, because make, curl, and wget are all installed.

**Comment by pcercuei** [comment]
What do `which make`, `which curl` and `which wget` return?

**Comment by godreborn** [comment]
> What do `which make`, `which curl` and `which wget` return?

![image](https://github.com/user-attachments/assets/7206c5cd-960b-4949-88bb-89cd8101a354)

works fine on ubuntu 22.04.  I told falco about the problem, and he wasn't sure what could be going wrong.

**Comment by pcercuei** [comment]
@godreborn ok, could you now run:
`command -v make curl wget`
`echo $SHELL`

**Comment by godreborn** [comment]
this?  
![image](https://github.com/user-attachments/assets/5242ba48-5681-48cd-be6c-a5bab0ca32e4)

**Comment by pcercuei** [comment]
Ok, that's strange. the Makefile complains that you don't have the `command` built-in shell command, but you clearly have it.

**Comment by godreborn** [comment]
> Ok, that's strange. the Makefile complains that you don't have the `command` built-in shell command, but you clearly have it.

yeah, ubuntu 22.04 has no problems.  ubuntu 20.04 originally had no issues, but I tried to update, to add cmake mostly, and it didn't work, unless there's a way to update an install of the os.

**Comment by darcagn** [comment]
I've actually seen this before, I believe it was a Debian VM on an older version, perhaps Ubuntu. It was stock, I had the dependencies, and everything was normal with the shell, environment, paths, etc.. I couldn't figure out what was up so I just used a newer version which worked. Very bizarre.

**Comment by pcercuei** [comment]
We could maybe use `which` instead of `command`?

**Comment by godreborn** [comment]
so what should I do?  thank you all for responding.

**Comment by pcercuei** [comment]
Please  try with https://github.com/KallistiOS/KallistiOS/pull/863

**Comment by godreborn** [comment]
thanks!  I'll try it.  I thought my distro was messed up somehow.  mine is almost 200GBs in size as wslv2, as I've installed and compiled over 500 things in it, so I thought maybe something had gotten corrupted somewhere.

**Comment by godreborn** [comment]
it seemed to work.  I did get an error with one of the kos-ports.  

![image](https://github.com/user-attachments/assets/297080d8-d676-4c8d-b767-547c8b13ec3b)

it didn't affect it building though, and I remember getting an error on ubuntu 22.04 as well, that was supposed to have been fixed.  I don't remember if it was the same error, but I think falco told me that the ubuntu 22.04 one was okay.

**Comment by godreborn** [comment]
![image](https://github.com/user-attachments/assets/dcb5812f-01b7-42f8-a0b7-cedddc7fe9f9)

here's the one that error'd.  I'm pretty sure it's the same one.

**Comment by godreborn** [comment]
okay, this also error'd with the kos examples:

![image](https://github.com/user-attachments/assets/bc3f909c-581c-416e-a58e-69711d878724)

**Comment by pcercuei** [comment]
Yeah jimtcl is broken, and it looks like some examples are too.

**Comment by godreborn** [comment]
okay, thanks.  I'll close this comment, since it's been resolved.  thank you all your time and help.
